vajram/vajram-codegen/src/main/groovy/com/flipkart/krystal/vajram/VajramPlugin.groovy

```diff
@@ -60,7 +57,8 @@ class VajramPlugin implements Plugin<Project> {
                 VajramCodeGenFacade.codeGenVajramImpl(
                         project.sourceSets.main.java.srcDirs,
                         compiledMainDir,
-                        mainGeneratedSrcDir)
+                        mainGeneratedSrcDir,
+                        project.configurations.compileClasspath)
```

Comment:
Should this be project.tasks.compileJava.compileClasspath?

From ChatGPT:

In gradle what is the difference between project.configurations.compileClasspath and project.tasks.compileJava.compileClasspath?

In Gradle, project.configurations.compileClasspath and project.tasks.compileJava.compileClasspath are two different ways to specify the classpath for the Java compilation process.

project.configurations.compileClasspath is a configuration object that defines the dependencies needed to compile the project's source code. It is used to specify the classpath for the compileJava task and other tasks that need to access the compiled classes. This configuration can be modified to include additional dependencies that are required for the project.

On the other hand, project.tasks.compileJava.compileClasspath is a property that specifies the classpath for the compileJava task only. This property is automatically set to the compileClasspath configuration by default, but it can be overridden if needed.

So, the main difference between them is that project.configurations.compileClasspath is a configuration object that can be modified to include additional dependencies, whereas project.tasks.compileJava.compileClasspath is a property that is specific to the compileJava task and cannot be modified globally.

Reply:
We can choose the one which better suits us. In case of test, I guess we have no other option but to use project.tasks.testCompileJava.compileClasspath -> I was thinking we should keep it consistent

Reply:
updated the classpath.